.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -55,11 +55,6 @@ Note: Adding GitHub labels is only possible for contributors with write access.
 
 - [ ] If known, an appropriate milestone has been selected; otherwise the `Triage` milestone is set.
 - [ ] Use the `major_change` label if your change either has a major impact on the code base, is impacting multiple teams or is changing important well-established internals of the Agent. This label will be use during QA to make sure each team pay extra attention to the changed behavior. For any customer facing change use a releasenote.
-- [ ] A [release note](https://github.com/DataDog/datadog-agent/blob/main/docs/dev/contributing.md#reno) has been added or the `changelog/no-changelog` label has been applied.
-- [ ] Changed code has automated tests for its functionality.
 - [ ] Adequate QA/testing plan information is provided. Except if the `qa/skip-qa` label, with required either `qa/done` or `qa/no-code-change` labels, are applied.
 - [ ] At least one `team/..` label has been applied, indicating the team(s) that should QA this change.
-- [ ] If applicable, docs team has been notified or [an issue has been opened on the documentation repo](https://github.com/DataDog/documentation/issues/new).
-- [ ] If applicable, the `need-change/operator` and `need-change/helm` labels have been applied.
-- [ ] If applicable, the `k8s/<min-version>` label, indicating the lowest Kubernetes version compatible with this feature.
```

Comment:
@amenasria Could we link the contrib doc here as well (hidden or not doesn't matter)? My understanding was that we'd do that from here rather than from the check output to prevent an extra feedback cycle.

Reply:
👋 Hey Srdjan ! Yes I'll do this, this PR is a WIP hence just some deletions for now, but I will add a link to the contrib doc.

I will also keep part of the reviewer checklist but it will become a contributor checklist and it might be a dropdown.